README.md

```diff
@@ -86,6 +86,14 @@ Danmaku can be emitted by calling `emit` method.
 damoo.emit({ text: "Damoo is awesome!", color: "#f49" });
 ```
 
+Add hyper link if you want to let the danmaku clickable
+
+加入超链接使弹幕能够点击跳转
+
+```javascript
+damoo.emit({ text: "Damoo is awesome!", color: "#f49", link : "script/Damoo.js"});
```

Comment:
I cannot get your point, maybe the hyper link is wrong? or you just don't want this function? Also I'm Chinese...